core/src/main/java/org/eclipse/hono/util/CredentialsResult.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2017 Bosch Software Innovations GmbH.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Bosch Software Innovations GmbH - initial creation
+ */
+
+package org.eclipse.hono.util;
+
+import io.vertx.core.json.JsonObject;
+
+/**
+ * A container for the result returned by Hono's credentials API.
+ *
+ */
+public final class CredentialsResult {
```

Comment:
This seems to be the same class as `RegistrationResult`. I would like to (later) refactor Credentials and Device Registration to use the same (generic) class for conveying results.